Sources/StreamChatUI/CommonViews/PlayPauseButton/PlayPauseButton.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+open class PlayPauseButton: MediaButton {
```

Comment:
This one also needs documentation, we try to add documentation to all new public views (Even if the name of the view is self explanatory)